test/puppetlabs/puppetdb/reports_test.clj

```diff
@@ -5,13 +5,17 @@
             [puppetlabs.puppetdb.testutils.reports
              :refer [munge-example-report-for-storage]]))
 
-(let [report (munge-example-report-for-storage (:basic reports))]
+(let [report (munge-example-report-for-storage (:basic reports))
+      report-missing-noop (dissoc report :noop)]
```

Comment:
Oh I see @wkalt's point now, in this test, we're testing its removal, not that its null. I think we could test for both cases, it equates to the same thing I think.